connector/MongoDBAtlasODBC.unit-test.query.pq

```diff
@@ -1,281 +1,349 @@
 section MongoDBAtlasODBCUnitTests;
-shared UnitTests = 
-[
-    facts = 
-    {
-        Fact("GetConnectionString - Set URI option",
-          [ DRIVER = "ADF_ODBC_DRIVER", DATABASE = "db", URI = "mongodb://hostname/admin?ssl=true" ],
-          GetConnectionString("mongodb://hostname/admin?ssl=true", "db")
+
+shared UnitTests = [
+    facts = {
+        Fact(
+            "GetConnectionString - Set URI option",
+            [
+                DRIVER = "ADF_ODBC_DRIVER",
+                DATABASE = "db",
+                URI = "mongodb://hostname/admin?ssl=true",
+                APPNAME = "powerbi-connector"

```

Comment:
@bucaojit I added this to the test so it'd pass. The rest of the changes are all from auto format. I've noticed that we have differing format settings. Might be worth syncing to avoid changes like this going forward.